src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1428,7 +1428,8 @@ static bool addTriggeredSubscriptions_noCache
   DBClientBase* connection = getMongoConnection();
   if (collectionQuery(connection, collection, query, &cursor, &errorString) != true)
   {
-    alarmMgr.dbError(errorString);
+    // FIXME P10: collectionQuery already calls dbMgr in the case of problem. Redundant alarmMsg.dbError() ?
```

Comment:
Detailed 'explanation' only in one of these redundant logging cases.

NTC, except removing these temporary lines